src/add/observable/bindCallback.ts

```diff
@@ -3,4 +3,8 @@ import {BoundCallbackObservable} from '../../observable/BoundCallbackObservable'
 
 Observable.bindCallback = BoundCallbackObservable.create;
 
-export var _void: void;
\ No newline at end of file
+declare module '../../Observable' {
+  module Observable {
```

Comment:
why not `namespace`  instead of `module`?

Reasoning: the key word module in these two lines mean two things, and it can be confusing. namespace would be clearer.